### PR TITLE
Change Cleared and Please Do Other Content role colors

### PR DESCRIPTION
**Pull request by ghost**
Changed all Cleared Roles and Please Do Other Content.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -592,6 +592,7 @@ guilds:
         color: 0xEAC645
         hoist: true
   encounters:
+  # Changed initial cleared role colors
```
Comment (brtran4) [inline config.yaml]: typically when we add comments, we describe the functionality/context for the code that the comment is attached to. but this seems like a changelog, so we won't need this

File: internal/clearingway/relevant-reptition-roles.go [hunk 2]
```diff
@@ -6,9 +6,10 @@ import (
 
 func RelevantRepetitionRoles(encs *Encounters) *Roles {
 	roles := &Roles{Roles: []*Role{
+		// Changed "Please Do Other Content" inital color - Claire
```
Comment (brtran4) [inline internal/clearingway/relevant-reptition-roles.go]: same as the other comment

**Comment by brtran4** [comment]
so how version control in code works is that every line change is connected to some commit. we can look at a commit hash (in your case it's 55b9330b33eee34ae87145bab8f0327293888c7e) and see who made it, what the description is, and what they did. for example:

Screenshot 2024-05-26 at 4 11 20 PM

**Comment by brtran4** [comment]
nit: change the title of the commit to be in the imperative voice aka "Change more role colors"

**Review by brtran4 — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 1 above)
Comment (ghost) [reply config.yaml]: fixed

**Review by brtran4 — APPROVED** [review]
let's make sure that we update the PR title to be in the imperative voice. same with #19

**Pull request merged**